node_modules/oae-core/setpermissions/js/setpermissions.js

```diff
@@ -141,8 +173,16 @@ define(['jquery', 'oae.core'], function($, oae) {
             // Trigger and bind an event that will ask for the page context
             $(document).on('oae.context.send.setpermissions.' + uid, function(ev, data) {
                 pageContext = data;
-                // Immediately pass the default permissions summary back to the widget caller
-                setUpAutoSuggest(savePermissionsChange);
+
+                // If the resource is inside of a folder, get the folder members to add them to the resource
+                getFolderMembers(pageContext.id, function(err) {
```

Comment:
Let's have this function returning the members as part of the callback, and then pass it as an argument to `setUpAutoSuggest`. Nice and easy.

Reply:
It doesn't work when I do that :/ Should I let the code like this or should I push an unworking code to debug it together ?